modules/troubleshooting/pages/index.adoc

```diff
@@ -0,0 +1,170 @@
+include::{mod-loc}attributes.adoc[]
+
+//Category: skupper-troubleshooting
+// Type: assembly
+
+[id="troubleshooting"] 
+= Troubleshooting a {service-network}
+
+[role="system:abstract"]
+Typically, you can create a {service-network} without referencing this troubleshooting guide.
+However, this guide provides some tips for situations when the {service-network} does not perform as expected.
+
+See xref:common-problems[] if you have encountered a specific issue using the `skupper` CLI.
+
+A typical troubleshooting workflow is to check all the sites, and create debug tarballs.
+
+// Type: procedure
+[id="checking-sites"] 
+== Checking sites
+
+[role="system:abstract"]
+Using the `skupper` command-line interface (CLI) provides a simple method to get started with troubleshooting {skupper-name}.
+
+.Procedure
+
+. Check the site status:
++
+--
+----
+$ skupper status --namespace west
+
+Skupper is enabled for namespace "west" in interior mode. It is connected to 2 other sites. It has 1 exposed services.
+----
+
+The output shows:
+
+* A site exists in the specified namespace.
+* A link exists to two other sites.
+* A service is exposed on the {service-network}, and is accessible from this namespace.
+
+--
+
+. Check the link status:
++
+--
+----
+$ skupper link status --namespace east
+
+Link link1 is active
+----
+
+A link exists from the specified site to another site, meaning a token from another site was applied to the specified site.
```

Comment:
```suggestion
A link exists from the specified site to another site, meaning a token from another site applies to the specified site.
```
Passive voice.

Reply:
doesn't work, need past tense, and in this case we don't know who did it... so leaving as is